CMakeLists.txt

```diff
@@ -1,12 +1,35 @@
 cmake_minimum_required(VERSION 3.15)
-set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+
+set(VCPKG_LIBRARY_LINKAGE static)
+include(${CMAKE_SOURCE_DIR}/cmake/ezvcpkg/ezvcpkg.cmake)
+ezvcpkg_fetch(
+    REPO jherico/vcpkg
+    # Current release has a bug in the cmake config for ktx, see https://github.com/microsoft/vcpkg/issues/30787
+    COMMIT a9f012790cf5f9be7993b99998cab106a94df9bd
```

Comment:
This will be replaced with no `REPO` declaration and the appropriate commit if and when my vcpkg PR is merged.  Right now it's building directly from my fork with the changes.